game.go

```diff
@@ -45,6 +45,7 @@ var (
 
 type Game struct {
 	gameTime      *mathf.GameTime
+	ticker        *mathf.Ticker
```

Comment:
This creates a memory leak as we never clear out the tickers for popped scenes.
Need to find some other way to handle "global" game values that are really tied to scenes, another example could be a content manager that we create right now directly in the scene.

Reply:
fixing this with a scene context later